RIS.js

```diff
@@ -6968,6 +6967,82 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "import",
+		"input": "TY  - JOUR\r\nTI  - Using focus groups to adapt ethnically appropriate, information-seeking and recruitment messages for a prostate cancer screening program for men at high risk\r\nT2  - Journal of the National Medical Association\r\nVL  - 100\r\nIS  - 6\r\nSP  - 674\r\nEP  - 682\r\nPY  - 2008\r\nDO  - 10.1016/S0027-9684(15)31340-7\r\nAU  - Bryan, C.J.\r\nAU  - Wetmore-Arkader, L.\r\nAU  - Calvano, T.\r\nAU  - Deatrick, J.A.\r\nAU  - Giri, V.N.\r\nAU  - Bruner, D.W.\r\nN1  - Cited By :10\r\nN1  - Export Date: 13 July 2022\r\nER  - \r\n\r\nTY  - JOUR\r\nT2  - Prostate Cancer Mortality Statistics 2015\r\nPY  - 0000\r\nN1  - Cited By :1\r\nN1  - Export Date: 13 July 2022\r\nER  - \r\n\r\n",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Using focus groups to adapt ethnically appropriate, information-seeking and recruitment messages for a prostate cancer screening program for men at high risk",
+				"creators": [
+					{
+						"lastName": "Bryan",
+						"firstName": "C.J.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Wetmore-Arkader",
+						"firstName": "L.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Calvano",
+						"firstName": "T.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Deatrick",
+						"firstName": "J.A.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Giri",
+						"firstName": "V.N.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Bruner",
+						"firstName": "D.W.",
+						"creatorType": "author"
+					}
+				],
+				"date": "2008",
+				"DOI": "10.1016/S0027-9684(15)31340-7",
+				"issue": "6",
+				"pages": "674-682",
+				"publicationTitle": "Journal of the National Medical Association",
+				"volume": "100",
+				"attachments": [],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Cited By :10</p>"
+					},
+					{
+						"note": "<p>Export Date: 13 July 2022</p>"
+					}
+				],
+				"seeAlso": []
+			},
+			{
+				"itemType": "journalArticle",
+				"creators": [],
+				"date": "0000",
```

Comment:
Do we even want to save these 0000 dates, though? Might be better to just skip all-zero date segments.

Reply:
I guess there's no upside to the 0000 -- I based keeping it on the MM/DD case above, but that's not actually the same since we add the year in order to keep the other info there. I'll look at removing it